### ISSUE TITLE
Unable to resolve module @unimodules/core with Expo SDK 43 expo-constants "~11.0.1"

### ISSUE THREAD

**Issue by koch-aivy** [bugsnag/bugsnag-js#1563]
### Describe the bug
Unable to resolve module @unimodules/core from /path-to-app/node_modules/@bugsnag/expo/node_modules/expo-constants/build/Constants.js

Constants.js tries to import from @ unimodules/core, which is deprecated from expo SDK 43 (e.g. described here, everything was moved to expo-modules-core) https://blog.expo.dev/whats-new-in- expo-modules-infrastructure-7a7cdda81ebc
The lib seems to use an older version of expo-constants, because in e expo-constants version "~ 12.1.1" it already gets imported from expo-modules-core. what options do I have here? 

### Steps to reproduce
1. Update expo sdk 43 
2. Install bugsnag
3. expo install expo-constants

### Environment
* Bugsnag version:@bugsnag/expo": "^7.13.2",  "@bugsnag/js": "^7.13.2", "@bugsnag/plugin-react": "^7.1.0",
* Browser framework version (if any):
    * react-native: "0.64.2",
* Device (e.g. iphonex): iPhone 12 simulator

<details><summary>Error messages:

Unable to resolve module @unimodules/core from /Users/leonie/Repos/aivy-multi-app/node_modules/@bugsnag/expo/node_modules/expo-constants/build/Constants.js: @unimodules/core could not be found within the project or in these directories:
  node_modules/@bugsnag/expo/node_modules/expo-constants/node_modules
  node_modules/@bugsnag/expo/node_modules
  node_modules

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*
> 1 | import { CodedError, NativeModulesProxy } from '@unimodules/core';
    |                                                 ^
  2 | import { Platform, NativeModules } from 'react-native';
  3 | import { AppOwnership, ExecutionEnvironment, UserInterfaceIdiom, } from './Constants.types';
  4 | import ExponentConstants from './ExponentConstants';

</summary>

```

```
</details>

**Comment by kevinavery** [comment]
Thanks for opening this issue - I'm seeing the same error in Xcode Logs using Expo's EAS build system.

**Comment by dpolivy** [comment]
I am also having Expo SDK 43 build errors with Bugsnag due to a dependency on an old version of `expo-crypto`:

````
Couldn't install: EXCrypto » UMCore. Ensure the CocoaPod "UMCore" is installed in your project, then run `npx pod-install` to try again.
````

Issue stems from this:
https://github.com/bugsnag/bugsnag-js/blob/2503f326a6f10d014b73ac87b8173ea18edef338/packages/delivery-expo/package.json#L21

**Comment by yousif-bugsnag** [comment]
Hi @koch-aivy, as noted above we're currently working on Expo 43 support under #1556. This should be available in the next release.

**Comment by luke-belton** [comment]
Hi @koch-aivy - the [fix](https://github.com/bugsnag/bugsnag-js/pull/1566) for this was released in [`v7.13.3`](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.13.3) 🎉
